conf.py

```diff
@@ -34,7 +34,7 @@
 author = 'This page is maintained by the Flux community.'
 
 # The full version, including alpha/beta/rc tags
-release = '0.13.0'
+release = '0.19.0'
```

Comment:
We should probably delete this line.  We did so in flux-core, and I don't think this is used anywhere.  @gonsie, can you confirm/do you agree?

If we don't delete it, we should set it to `0.1.0` or `0.0.0` since flux-docs will be versioned independently from flux-core and flux-sched.

Reply:
Yes, too much churn if we have to update this each time. The version number shows up in the browser window and it was confusing. 

Do we need a way to sync the html doc version with flux-core version though?

Reply:
@SteVwonder and @gonsie: just recommend what you need with the version number, I will adjust the PR.

Reply:
> Do we need a way to sync the html doc version with flux-core version though?

From previous conversations in coffee time and issues, I think the conclusion we came to is:
"Each repo has it’s own documentation and that gets tagged and released along with the code, but the high-level “meta” documentation has it’s own versioning that is divorced from any particular sub-packages/repos versioning."

So the top-level docs can refer to either the latest version of flux-core docs or a specific tag (if necessary) via Sphinx links.  But the versioning of flux-docs itself is entirely independent.  We still need to come up with conventions for how and when we want to tag flux-docs.

Reply:
> @SteVwonder and @gonsie: just recommend what you need with the version number, I will adjust the PR.

What happens if we just wholesale delete the line of code like was done in flux-core? I think it should just not version anything, and we will just get `latest`.  Which seems fine to me for now until we come up with a convention for tagging and releasing flux-docs.  Thoughts?

Reply:
> So the top-level docs can refer to either the latest version of flux-core docs or a specific tag (if necessary) via Sphinx links. But the versioning of flux-docs itself is entirely independent. We still need to come up with conventions for how and when we want to tag flux-docs.

Very wise decision! Thanks.